hw/ip/prim/lint/prim_fifo.waiver

```diff
@@ -10,21 +10,9 @@ waive -rules {ONE_BIT_MEM_WIDTH} -location {prim_fifo_sync.sv} -msg {Memory 'gen
 waive -rules {INPUT_NOT_READ} -location {prim_fifo_sync.sv} -regexp {Input port '(clk_i|rst_ni)' is not read from, instance.*Depth=0\)} \
       -comment "In passthrough mode, clk and reset are not read form within this module"
 
-
-waive -rules {ASSIGN_SIGN} -location {prim_fifo_async.sv} -msg {Signed target 'i' assigned unsigned value 'PTR_WIDTH - 3'} \
```

Comment:
Is a similar waiver still required? The solution really is to introduce an explicit cast in the RTL, and the explanatory comment is correct that a signed type must be used here. Or the loop could emit dec_tmp[i-1] instead.

Reply:
Good point. Let's just make it more sensible: I'll force-push with a cleaner solution in a sec.